libcontainerd/types_windows.go

```diff
@@ -67,6 +67,11 @@ type CredentialsOption struct {
 	Credentials string
 }
 
+// DNSOption is a CreateOption that provides container specific DNS settings
```

Comment:
Yes it can be. I just thought It would be better to have it as a separate option. I will combine them